src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -1434,7 +1434,9 @@ private void initActions() {
         twoEntriesOnlyActions.addAll(Arrays.asList(mergeEntries));
 
         atLeastOneEntryActions.clear();
-        atLeastOneEntryActions.addAll(Arrays.asList(downloadFullText, lookupIdentifiers));
+        atLeastOneEntryActions.addAll(Arrays.asList(downloadFullText));
+        atLeastOneEntryActions.clear();
```

Comment:
I don't understand these changes: you first add the download action but then clear the list right after.